### PR TITLE
Feature/automate main test cases

### PR DESCRIPTION
**Pull request by goncer**

### CODE REVIEW

**Comment by goncer** [comment]
I think that is big enough. Will add more tests in a separate ones.

File: utils/utils.ts [hunk 1]
```diff
@@ -11,7 +11,11 @@ import { getAccountJSON } from "./frontend/utils/Helper";
 
 export const MGA_ASSET_ID = new BN(0);
 export const MGA_ASSET_NAME = "MGA";
+export const ETH_ASSET_ID = new BN(1);
```
Comment (eddyjlsh) [inline utils/utils.ts:14]: It could be worth moving these asset names and IDs to an external config file

**Review by eddyjlsh — APPROVED** [review]
LGTM :)

**Review by Andkoo — APPROVED** [review]

File: utils/utils.ts (hunk 1 above)
Comment (goncer) [reply utils/utils.ts]: done!
Created Constants.ts file.

**Pull request merged**